.github/workflows/test.yml

```diff
@@ -106,9 +106,9 @@ jobs:
           # newest Android version
           saucectl run espresso -c "" --name "From Github Actions" --app example-app-debug.apk --testApp ${{ matrix.test-apk }}  --emulator name="Android GoogleApi Emulator,platformVersion=12.0" --region us-west-1
           # oldest Android version
-          saucectl run espresso -c "" --name "From Github Actions" --app example-app-debug.apk --testApp ${{ matrix.test-apk }}  --emulator name="Android GoogleApi Emulator,platformVersion=6.0" --region us-west-1
+          saucectl run espresso -c "" --name "From Github Actions" --app example-app-debug.apk --testApp ${{ matrix.test-apk }}  --emulator name="Android GoogleApi Emulator,platformVersion=8.0" --region us-west-1
```

Comment:
I think the emulator is ok with 6.0? I only hit an error with physical devices.

Reply:
Minimum supported device in VDC is Android 5.

Reply:
I think it can be dropped any time since we want to remove legacy support for old os